packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -347,7 +358,7 @@ export const HelpCenterContactForm = () => {
 						is_chat_overflow: overflow,
 						source: 'source_wpcom_help_center',
 						blog_url: supportSite.URL,
-						ai_chat_id: gptResponse?.answer_id,
+						ai_chat_id: wapuuFlow ? supportChatId ?? '' : gptResponse?.answer_id,
```

Comment:
```suggestion
						ai_chat_id: wapuuFlow ? ( supportChatId ?? '' ) : gptResponse?.answer_id,
```
Just to improve the readability a bit.

Reply:
I can't do it, when saving the linter removes the parenthesis :D

Reply:
Hah, well ok then, thanks for trying 😄